### DISCUSSION TITLE
Is it possible to change the video file without changing the metadata?

### DISCUSSION THREAD

**Discussion by sbdallas** [stashapp/stash#5506] [Q&A] [answered]
Let's say I have a file, an old DVD5 rip from 1992.  I get an AI upscaled version of the file.  I want to replace the file without messing up my entry.  I already thought about renaming the new file with the old file's name; the files have different extensions.  I don't want to lose the metadata because I had to enter it all manually.  Any help on this would be greatly appreciated.

**Comment by stg-annon** [comment, marked as answer]
stash can pick up a file as long as either the filename to changes or the file content changes **but not both at the same times** so couple of options 

1. rename the new file the exact same name as the old file and have stash scan it it will pick it up as a update of the old file you then rename it again to the correct file extension as they are likely different  and scan one more time

2. scan the new file and merge it into the existing entry in stash this is fairly straightforward but you will likely need to regenerate covers or previews as they are associated with the file oshash

**Reply by sbdallas** [reply]
That worked like a charm!  TY, friend!
